qt_binder/binder.py

```diff
@@ -568,6 +564,100 @@ def _on_trait_change(self, handler, name=None, remove=False,
                 for func, name in connectors:
                     func(self, name)
 
+    def _collect_renamings(self, binder_class):
+        """ Collect all of the renamings requested by Rename traits.
+        """
+        renamings = {}
+        # Find all requested renamings.
+        for name in dir(binder_class):
+            obj = getattr(binder_class, name)
+            if isinstance(obj, Rename):
+                renamings[obj.qt_name] = name
+        return renamings
+
+    def _add_qt_properties(self, binder_class, renamings, seen):
+        """ Add QtProperties.
+        """
+        meta_object = binder_class.qclass.staticMetaObject
```

Comment:
this method is simply lines 590-599 from master refactored into a method - no new logic has been added.